bigbone/src/integrationTest/kotlin/social/bigbone/TestConstants.kt

```diff
@@ -0,0 +1,22 @@
+package social.bigbone
+
+class TestConstants {
```

Comment:
Some of these constants seem unused. Especially, port definitions (443 for both REST and streaming APIs) are not used and different from the ports mentioned on the Testing wiki page (3000/4000). Is this correct?

Reply:
Ye, there are some unused constants. I'll double check. The way the Docker Compose setup works is basically this:
- Mastodon REST instance on port 3000 is started
- Mastodon Streaming instance on port 4000 is started
- Nginx is launched which accepts traffic on port 443 and forwards it to one of them above.

I've decided to go this way, as the recommendation for running Ruby locally is using SSL, as some stuff might not work properly if served via HTTP only. That's why Nginx is used as reverse proxy in front of them.

If a client requests something like /api/v2/streaming/... it goes to the streaming instance, anything else goes to the regular instance.

So for the integration tests, we should connect MastodonClient to port 443 for the moment. We can change this of course.